src/components/Header/index.js

```diff
@@ -4,15 +4,17 @@ import cs from "./styles.module.css"
 
 export default () => {
   return (
-    <header className={cs.header}>
-      <a href="https://mutualaidmamas.com/">
+      <header className={cs.header}>
+          <div className={cs.logo}>
```

Comment:
@dcalacci what editor are you using? We use [prettier](https://prettier.io/) for formatting. This looks very strange. with the enclosing `<div>` being indented more than its children??

Reply:
lol I'm in the middle of configuring a new machine. I use emacs and not all of my dev environment is set up yet, that was.... not automatically done, lol

Reply:
Emacs is hardcore.